xml/containers-build.xml

```diff
@@ -0,0 +1,171 @@
+  <title>SUSE-Internal Build Tools</title>
+
+  <sect2 xml:id="id-open-build-service">
+   <title>Open Build Service</title>
+   <para>
+    The <emphasis role="strong">Open Build Service</emphasis> (OBS) provides
+    free infrastructure for building and storing RPM packages including various
+    container formats. An important component of the OBS is also the
+    <emphasis role="strong">OBS Container Registry</emphasis>. The
+    <link xlink:href="https://registry.opensuse.org/cgi-bin/cooverview">OBS
+    Container Registry</link> provides a detailed listing of all container
+    images built by the OBS complete with command line code to pull the images
+    into your local docker environment. The OBS &opensuse;
+    <link xlink:href="https://build.opensuse.org/image_templates">container
+    image templates</link> can be modified to specific needs and are the
+    easiest way to create your own container branch. Container images can
+    either be built with <emphasis role="strong">native Docker tools</emphasis>
+    - based on an existing image using a Dockerfile. Or they can be built from
+    scratch using the <emphasis role="strong">&kiwi; imaging
+    solution</emphasis>.
+   </para>
+   <para>
+    Instructions on how to build images on OBS can be found
+    <link xlink:href="https://openbuildservice.org/2018/05/09/container-building-and-distribution/">here</link>.
+   </para>
+  </sect2>
+
+  <sect2 xml:id="id-kiwi">
+   <title>&kiwi;</title>
+   <para>
+    <emphasis role="strong">KIWI Next Generation</emphasis> is a multi purpose
+    imaging solution. Beside container images, classical installation ISOs and
+    images for virtual machines &kiwi; can also build images that boot via PXE
+    or Vagrant boxes. The basis is an image XML description. Such a description
+    is represented by a directory which includes at least one
+    <literal>config.xml</literal> or <literal>.kiwi</literal> file and may as
+    well include other files like scripts or configuration data. The image
+    creation process with &kiwi; is automated and does not require any user
+    interaction. Any information required for the image creation process is
+    provided by the primary configuration file: <literal>config.xml</literal>.
+    In addition, the image can optionally be customized using the
+    <literal>config.sh</literal> and <literal>images.sh</literal> scripts.
+   </para>
+   <note>
+    <para>
+     It is important to distinguish between KIWI NG (currently version 9.20.9)
+     and it’s unmaintained legacy versions (7.x.x or older), now called
+     <link xlink:href="https://doc.opensuse.org/projects/kiwi/doc/">KIWI
+     Legacy</link>.
+    </para>
+   </note>
+   <para>
+    For specific information on how to install &kiwi; and use it to build
+    images, see the <link xlink:href="http://osinside.github.io/kiwi/">KIWI
+    documentation</link>. A collection of example image descriptions can be
```

Comment:
We have this documentation at documentation.suse.com/kiwi/ now. Maybe link there instead (though both locations will continue to coexist).